I'm sorry, but an uncaught exception occurred.

After initialization, but before game start.
  File "renpy/common/00start.rpy", line 83, in _init_language
    renpy.change_language(language)
  File "game/screens.rpy", line 578, in <module>
    background frame("#4b0a52ff", 25, 25)
NameError: name 'frame' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\Megan\Desktop\renpy-8.0.3-sdk\renpy\bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "C:\Users\Megan\Desktop\renpy-8.0.3-sdk\renpy\main.py", line 660, in main
    run(restart)
  File "C:\Users\Megan\Desktop\renpy-8.0.3-sdk\renpy\main.py", line 76, in run
    renpy.translation.init_translation()
  File "C:\Users\Megan\Desktop\renpy-8.0.3-sdk\renpy\translation\__init__.py", line 583, in init_translation
    renpy.store._init_language() # type: ignore
  File "renpy/common/00start.rpy", line 83, in _init_language
    renpy.change_language(language)
  File "C:\Users\Megan\Desktop\renpy-8.0.3-sdk\renpy\translation\__init__.py", line 667, in change_language
    new_change_language(tl, language)
  File "C:\Users\Megan\Desktop\renpy-8.0.3-sdk\renpy\translation\__init__.py", line 629, in new_change_language
    i.apply()
  File "C:\Users\Megan\Desktop\renpy-8.0.3-sdk\renpy\ast.py", line 2907, in apply
    value = renpy.python.py_eval(expr)
  File "C:\Users\Megan\Desktop\renpy-8.0.3-sdk\renpy\python.py", line 1092, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Megan\Desktop\renpy-8.0.3-sdk\renpy\python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/screens.rpy", line 578, in <module>
    background frame("#4b0a52ff", 25, 25)
NameError: name 'frame' is not defined

Windows-10-10.0.22621 AMD64
Ren'Py 8.0.3.22090809
Voices From Beyond 1.0
Fri Mar 10 21:59:30 2023
